h/static/styles/partials-v2/_form-input.scss

```diff
@@ -80,16 +80,31 @@
     right: $h-padding;
 
     color: $grey-5;
- }
+  }
+
+  .form-input__hint-icon {
+    display: none;
+    position: absolute;
+    top: 10px;
+    left: 55px;
+    cursor: help;
+
+    .env-js-capable & {
+      display: block;
+    }
```

Comment:
There is one extra case you need to deal with here which is where the browser is JS capable but loading scripts fails or takes too long. You can simulate this by appending `__env__=js-timeout` at the end of the URL. Search for uses of `env-js-timeout` to see how we handle this elsewhere in the code.